### ISSUE TITLE
Docker set up doesn't work for me

### ISSUE THREAD

**Issue by caprenter** [resonatecoop/stream2own#232]
### Operating system
Ubuntu 20.04

### Browser name and version
Firefox 99

### Expected behavior
I'm expecting to see a running streaming site on http://localhost:8080 when I run docker-compose up -d 

### Actual behavior
The build fails in a couple of places. When I fix those I don't seem to have a persistent image that stays up

### Steps to reproduce behavior
Clone the repo
cd stream
docker-compose up -d
Build will fail.
Two steps to help it along:
Error: missing .env file
in /embed create a .env file from the .env.example (I did not change any of the contents)
Error: missing directory
mkdir dhparam/ (for the certificates to sit in)

I'm no docker expert, so sorry if I've missed the point somewhere.
If the docker build is something that should be easy for development, then I'm happy to help document it.

**Comment by auggod** [comment]
The docker compose config is up to you. I think, after adding the missing .env and creating the dhparam dir, it should be working.

**Comment by caprenter** [comment]
Ok - thanks. I'll investigate further.

**Comment by auggod** [comment]
Maybe the changes I have just made to the docker-compose.yml file will help.

**Comment by caprenter** [comment]
Yes that does help. Thanks! I am relatively new to using docker, so maybe I should have spotted something odd with those volumes, but I didn't!

So to confirm. I'm pretty sure my steps are:

Clone the repo and cd into it:
`git@github.com:resonatecoop/stream.git`
`cd stream`

Create 2 .env files
 `cp beta/.env.example beta/.env`
`cp embed/.env.example embed/.env`

Create a directory for the certbot certificates
`mkdir dhparam/`

then run 

`docker-compose up -d`

When completed you will have 4 containers, stream_beta_1, stream_embed_1, webserver, certbot

And the services are at:
`http://localhost:8080`
 `http://localhost:8081`

**Comment by caprenter** [comment]
Just an update. 

I think the above gives me a version of the site that runs, but that cannot be developed easily. 
(i.e. if I change a file, it doesn't update on the fly)
I've been googling around and also seen posts about how and when the npm install command is run, and setting up volumes outside the container so that node can watch changes (or the use of  the 'nodemon' module to watch for changes)

So I have a friend that I think I can persuade to help me figure out a good working dev solution with docker.
If anyone else already does this and can point me to instructions, then that would save me some time (and from having to learn anything - ha!), but I'll happily come back and share if I get there before that.

I'm actually running stuff without docker simply and easily, so I can try to develop without it - but I quite like the docker stuff for knowing I have the same shared development environment as others.

Thanks for all your help folks.

**Comment by simonv3** [comment]
FWIW, in my experience doing automatic reloaded front-end web dev inside docker can be pretty slow.
